metricbeat/module/kubernetes/state_node/_meta/test/kube-state-metrics.expected

```diff
@@ -1,66 +1,90 @@
 [
 	{
-		"_namespace": "node",
-		"cpu": {
-			"allocatable": {
-				"cores": 3
+		"RootFields": null,
+		"ModuleFields": null,
+		"MetricSetFields": {
+			"_namespace": "node",
```

Comment:
I would assume with v2 no `_namespace` field should be needed anymore?

Reply:
I checked the code and it seems for prometheus modules namespace is used a bit different then I expected.